.gitignore

```diff
@@ -151,6 +151,7 @@ crashlytics-build.properties
 fabric.properties
 
 
+config.json
 token.txt
```

Comment:
We don't need a `token.txt` file anymore, right? Could we remove all references to it to avoid confusion?

Reply:
> We don't need a `token.txt` file anymore, right? Could we remove all references to it to avoid confusion?

I re-added it for historical purposes and so that people don't accidentally commit it again.